packit_service/constants.py

```diff
@@ -46,6 +46,10 @@
     "</tr>"
 )
 
+MSG_DOWNSTREAM_JOB_ERROR_ROW = (
+    "<tr>" "<td><code>{branch}</code></td>" '<td>See <a href="{url}">{url}</a></td>' "</tr>\n"
```

Comment:
```suggestion
    "<tr>" "<td><code>{branch}</code></td>" '<td>See <a href="{url}">{url}</a></td>' "</tr>\n"
```
I am a bit puzzled by all this double quotes, it is not the first time I see them, I think they are related with ruff?
Do you know why so many?  🙏🏻

Reply:
yes, you are right, I think this was changed after running pre-commit, for now adjusted it manually and will try to have a look into it more